### PR TITLE
quiz rule page

### PR DESCRIPTION
**Pull request by mehulsatardekar**
### **Added quiz rule page.**

 preview: https://deploy-preview-6--quizgrad.netlify.app/views/quiz-rule

quiz-rule

### CODE REVIEW

File: views/quiz-rule.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            <ul class="qg-rules list-number">
+                <li>This quiz consists of 5 multiple-choice questions.</li>
+                <li>To be successful with the quizzes, it’s important to be conversant with the topics.</li>
+                <li>Each multiple choice question has only one correct answer.</li>
+                <li> To win the quiz you need to score more than 70%.</li>
+                <li>To start, click the  <span class="label-text label-text-primary">Start</span>  button. When finished, click the <span class="label-text label-text-danger">Submit</span> button.</li>
+            </ul>
+        </div>
+
+
```
Comment (pradipgharge) [inline views/quiz-rule.html]: remove extra spacing here

**Review by pradipgharge — APPROVED** [review]
No improvements, code looks nice and good to merge

**Review by rahulk31 — APPROVED** [review]
Code is well structured and like the way you have written less explicit css and made most out of your own CL. Can be merged 👍

File: public/css/quiz-rule.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+}
+.img-text-left{
+    position: absolute;
+    top: 1.2rem;
+    right: -0.2rem;
+    color: var(--img-text);
+    font-weight: 500;
+    font-size: 1.2rem;
+    padding: 5px 0.75rem;
+    background: var(--img-banner);
```
Comment (manojsarna) [inline public/css/quiz-rule.css:31]: can use px values

File: public/css/quiz-rule.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+}
+
+.navbar-bg{
+    background: var(--navbar-bg);
+}
+
+.nav-zindex{
+    z-index: 9;
+}
```
Comment (manojsarna) [inline public/css/quiz-rule.css:16]: why zindex 9 ?

File: views/quiz-rule.html [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+          </ul>
+      </div>
+      
+  </nav>  
+
+
+    <main class="container">
+      <section class="flex flex-wrap flex-column mt-3 mb-3">
+        <div class="flex flex-between flex-align-item-center flex-wrap ">
+            <div class="flex flex-column mb-3 ">
```
Comment (manojsarna) [inline views/quiz-rule.html]: use spacing consistently

**Review by manojsarna — COMMENTED** [review]
overall looks good!!

you can merge after checking the comments!!

**Review by manojsarna — APPROVED** [review]
Overall looks good!!

File: views/quiz-rule.html (hunk 4 above)
Comment (j24m) [reply views/quiz-rule.html]: Yes same advice, use consistent indentation :)

File: public/css/quiz-rule.css (hunk 2 above)
Comment (mehulsatardekar) [reply public/css/quiz-rule.css:31]: > can use px values

@manojsarna as Vikram told in his session if it is more than 10px then use rem

**Review by j24m — APPROVED** [review]
Hello, First of all would like to appreciate the amount of effort you have put in your UI, looks amazing . Now coming back to the code , everything looks fine except for consistent indentation. Hard work reflects. <3 :)

File: public/css/quiz-rule.css (hunk 3 above)
Comment (mehulsatardekar) [reply public/css/quiz-rule.css:16]: > why zindex 9 ?

Basically, the image container was hiding the navbar component.

**Pull request merged**